### ISSUE TITLE
ads namespaced proto generates with "Googlead" namespace

### ISSUE THREAD

**Issue by samphippen** [googleapis/gapic-generator-ruby#137]
proto file: https://github.com/googleapis/googleapis/blob/master/google/ads/googleads/v1/services/campaign_service.proto

result:
```
module Google
  module Ad
    module Googlead
      module V1
        module Service
          ##
          # Service to manage campaigns.
          class CampaignServiceClient
```

namespace is specified as `option ruby_package = "Google::Ads::GoogleAds::V1::Services";`

**Comment by samphippen** [comment]
full generated file here: https://gist.github.com/samphippen/663759f55770957203ca6b0061726590

looks like on line 52 the ALL_SCOPES constant may need some tweaking
on line 24 the requires are referencing cloud and not ads namespaces
there are some other nascent references to cloud throughout the file.

**Comment by blowmage** [comment]
This should have been fixed in #128. I will try to repro today.

**Comment by samphippen** [comment]
this was due to a janked docker setup.
